package.json

```
@@ -11,9 +12,10 @@
   ],
   "scripts": {
     "build": "lerna run build",
+    "build:ci": "lerna run build --ignore='example-*'",
     "doctoc": "doctoc --no-title --maxlevel 3 README.md CONTRIBUTING.md packages/*/README.md",
     "website:storybook": "lerna run storybook:build",
```

Ran into this while testing locally. It appears that if the output directory (`website/stories`) doesn't exist before running the script, it will sometimes generate only one storybook, with the other one failing ([see log here](https://app.netlify.com/sites/instantsearchjs/deploys/63a1c74333215b000b787069#L624)).

Having an existing directory first seems to do the trick.

```suggestion
    "website:storybook": "mkdir -p website/stories && lerna run storybook:build",
```

---

Hm, I tried deleting the folder and it worked for me, I guess it's a race condition where it thinks it should create the folder but actually shouldn't.

Also odd there's an error but netlify didn't fail?